src/app/rundown-overview/components/rundown-overview/rundown-overview.component.scss

```diff
@@ -42,7 +42,14 @@
   }
 }
 
-.c-rundown-overview-title {
+.c-rundown-overview {
+  &__no-rundowns,
+  &__loading-rundowns {
+    text-align: center;
+  }
+}
+
+.c-rundown-overview__title {
```

Comment:
I must admit, I don't think we gain much value from using BEMIT.
`c-rundown-overview__` is redundant to add since Angular allready uses ViewEncapsulation to scope the CSS.
All CSS classes defined in `rundown-overview.component.css` will only be applied in html files that has been told to use this specific file. Currently, that is only the `rundown-overview.component.html`.

So we add a lot of "clutter" to our CSS names that we don't gain a benefit from.
https://angular.io/guide/view-encapsulation

Reply:
We need to make a decision for what we do here before we go too far in one direction, or stray in to many different ones. 
I can agree that the BEMIT convention might not have as much purpose in Angular as it would in other scenarios.

Reply:
I do see your point. And it might not be that it is as needed in Angular as it is for projects that do not use Shadow DOM. But the structure in the class names still give useful information. By looking at this single line, I have a lot of information about what the class is describing. I know that we are in the scope of the Rundown overview component and that we are targeting the title. I don't even have to look at the filename.

When debugging and going through the DOM, then these classes are also useful, in the same sense that the context is given. If the class was simply `.title` and I were to search for that in the entire project, there would be a lot to filter out before finding the correct one. So again, it informs the structure and context to the reader.

Another argument is that if we use BEMIT throughout our products, then we will have a structure to follow when writing SCSS/CSS. As you mentioned, `the right tool for the right job`, which is correct, but as I see it, the pros of having a structure/standard for how we do styling outweighs the possible redundancy it might bring for the minority of our products.

Reply:
I do like the fact that we can go from one project to another and then have a good idea of what to expect.
We just need to be carefull with that argument because that quickly becomes a slippery slope to use something that is not a good choice for a given project, because it's a good choice for three different projects.

Reply:
I like the point about specifically naming separation within the file as well as the ease of finding things when going through the DOM.